docs/technical-overview/feature_flags.md

```diff
@@ -0,0 +1,87 @@
+# Feature flags
+
+We sometimes employ feature flags to develop features that aren't fully formed
+at the moment of deployment. They can also be employed to roll out features in
+productions incrementally.
+
+Feature flags are meant to be temporary and part of a rollout plan resulting in
+their removal.
+
+## Creating a new feature flag
+
+To create a new feature flag, we use data update scripts so that they can be
```

Comment:
**(nitpick, non-blocking):** Should we maybe also maybe link to the data update scripts documentation here? you do explain how it works below so not critical but a nice to have I guess.

Reply:
good idea!